packages/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -30,14 +30,14 @@ import * as mongodb from 'mongodb';
 import * as shimmer from 'shimmer';
 
 /** MongoDBCore instrumentation plugin for OpenTelemetry */
-export class MongoDBCorePlugin extends BasePlugin<typeof mongodb> {
+export class MongoDBPlugin extends BasePlugin<typeof mongodb> {
   private readonly _SERVER_METHODS = ['insert', 'update', 'remove', 'command'];
   private readonly _CURSOR_METHODS = ['_next', 'next'];
 
-  private readonly _COMPONENT = 'mongodb-core';
+  private readonly _COMPONENT = 'mongodb-nodejs';
   private readonly _DB_TYPE = 'mongodb';
```

Comment:
is `mongodb` a valid db type? I would think it would be `document` or `schemaless` or something like that

Reply:
It is : https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-database.md